cmake/ParaViewModules.cmake

```diff
@@ -310,12 +383,6 @@ function(add_external_project_internal name)
       LD_LIBRARY_PATH "${ld_library_path}")
   endif ()
 
-  set(ldflags_argument)
-  list(APPEND ldflags_argument LDFLAGS "${ldflags}")
-  if (SKIP_LDFLAGS_FOR_BUILD)
-    set(ldflags_argument)
-    set(SKIP_LDFLAGS_FOR_BUILD PARENT_SCOPE)
-  endif()
```

Comment:
This block was needed to make scipy build correctly on Linux.  It hadn't migrated back to PV superbuild yet.